src/algebra/ordered_field.lean

```diff
@@ -104,174 +130,75 @@ div_mul_cancel b (ne_of_lt hc) ▸ mul_le_mul_of_nonpos_right h (le_of_lt hc)
 lemma div_le_of_mul_le_of_neg (hc : c < 0) (h : a * c ≤ b) : b / c ≤ a :=
 calc
    a   = a * c * (1 / c) : mul_mul_div a (ne_of_lt hc)
-    ... ≥ b * (1 / c)     : mul_le_mul_of_nonpos_right h (le_of_lt (one_div_neg_of_neg hc))
+    ... ≥ b * (1 / c)     : mul_le_mul_of_nonpos_right h (le_of_lt (one_div_neg.2 hc))
     ... = b / c           : eq.symm $ div_eq_mul_one_div b c
 
-lemma mul_lt_of_gt_div_of_neg (hc : c < 0) (h : a > b / c) : a * c < b :=
+lemma mul_lt_of_div_lt_of_neg (hc : c < 0) (h : b / c < a) : a * c < b :=
```

Comment:
The name of the lemma does not match the order of its assumptions. I don't know if the best fix is to change the name, or the order of the assumptions